Lib/test/libregrtest/runtest.py

```diff
@@ -194,7 +193,7 @@ def get_abs_module(ns: Namespace, test_name: str) -> str:
         return 'test.' + test_name
 
 
-def _runtest(ns: Namespace, test_name: str) -> TestResult:
+def _runtest(result: TestResult, ns: Namespace):
```

Comment:
Most of the functions in this file appear to have type annotations; I'd prefer it if we could keep to the existing style here and have a return annotation for this function